spec/mailers/previews/training_request_mailer_preview.rb

```diff
@@ -0,0 +1,11 @@
+class TrainingRequestMailerPreview < ActionMailer::Preview
+
+  def notify_facility_staff
+    # Is not null and not blank. Oracle does not support `where.not(traning_request_contacts: "")` for CLOBS.
+    products_with_contacts = Product.requiring_approval.where("LENGTH(training_request_contacts) > 0")
+    product = NUCore::Database.random(products_with_contacts)
+    user = NUCore::Database.random(User.all)
```

Comment:
Fancy.

Reply:
I've done this on other projects. In some projects we've even gone as far as to put a `sample`/`random` scope on ActiveRecord::Base so you could call `User.random`, but since we're just using this in mailer previews it didn't seem worth putting it that globally.

Reply:
👏